docs/components/_index.md

```diff
@@ -9,9 +9,22 @@ aliases:
   - "/components/"
 ---
 
-A _component_ represents a physical piece of electrical or electromechanical hardware in a machine.
+Viam provides support for a wide variety of hardware.
+A _component_ represents a physical piece of hardware in your {{< glossary_tooltip term_id="machine" text="machine" >}}, and the software that directly supports that hardware.
```

Comment:
Maybe more accurate?
```suggestion
A _component_ represents a physical piece of hardware in or attached to your {{< glossary_tooltip term_id="machine" text="machine" >}}, and the software that directly supports that hardware.
```

Reply:
If a machine is an organizational concept, I think "in" is correct? I would say "that is part of your machine" but that confuses people with the special word "part."

Reply:
maybe we should keep the organization concept as part of the glossary description then? Ok to leave as is. Consider the glossary edit

Reply:
okay putting org concept back into the glossary